src/monsters.cpp

```diff
@@ -186,6 +186,121 @@ void MonsterType::createLoot(Container* corpse)
 	corpse->startDecaying();
 }
 
+void MonsterType::clone(MonsterType* root, MonsterType* child, const std::string type) {
+	if (type == "attacks") {
+		if (!child->spellAttackList.empty()) {
+			for (spellBlock_t attacks : child->spellAttackList) {
+				root->spellAttackList.push_back(attacks);
```

Comment:
This is bad. You're copying owning raw pointers into another container which could result in the spell objects being deleted multiple times (e.g. on a reload of monsters or spells).